packages/example/webpack.config.js

```diff
@@ -27,14 +27,17 @@ module.exports = {
     new DefinePlugin({
       'process.env.MAINNET_URL': process.env.MAINNET_URL ? JSON.stringify(process.env.MAINNET_URL) : undefined,
       'process.env.LOCALHOST_URL': JSON.stringify(process.env.LOCALHOST_URL)
-    })
+    }),
+    new EnvironmentPlugin({
+      ...process.env,
```

Comment:
We should not do it that way, to make sure we're not leaking anything - change before potential merge.